src/polymer/polymer-element.ts

```diff
@@ -29,9 +29,12 @@ export interface BasePolymerProperty {
   observer?: string;
   observerNode?: estree.Expression|estree.Pattern;
   reflectToAttribute?: boolean;
-  configuration?: boolean;
-  getter?: boolean;
-  setter?: boolean;
+  /**
+   * True if the property is part of Polymer's element configuration syntax.
+   *
+   * e.g. 'properties', 'is', 'extends', etc
+   */
+  isConfiguration?: boolean;
```

Comment:
I commented above, but I don't like this flag so much. I'd rather just consider these protected. Is it used elsewhere?